templates/items/details/details-facility.hbs

```diff
@@ -9,12 +9,27 @@
     {{ formField fields.type.fields.subtype value=source.type.subtype choices=facilitySubtypes }}
 
     {{!-- Size --}}
-    {{ formField fields.size value=source.size choices=config.facilities.sizes }}
+    {{ formField fields.size value=source.size choices=config.facilities.sizes labelAttr="label" }}
 
     {{!-- Level --}}
+    {{#if (eq source.type.value "basic")}}
     {{ formField fields.level value=source.level }}
+    {{/if}}
 
     {{#if (eq source.type.value "special")}}
+    {{!-- Properties --}}
+    <div class="form-group split-group">
+        <label>{{ localize "DND5E.FACILITY.Properties" }}</label>
+        <div class="form-fields">
+
+            {{!-- Level --}}
+            {{ formField fields.level value=source.level label="DND5E.Level" localize=true classes="label-top" }}
+
+            {{!-- Order --}}
+            {{ formField fields.order value=source.order classes="label-top" options=orders.available blank="" }}
```

Comment:
Moved this back under a 'properties' split group alongside 'level'. We have the order that the facility is capable of executing, which is what this is, and it's an intrinsic property of the facility. And we also have the order that the facility is currently executing, which might be the aforementioned order, the Maintain order, or nothing. That order is now part of the 'progress' schema and I put it inside the 'orders' fieldset alongside the progress fields.